diag_manager/fms_diag_object.F90

```diff
@@ -226,7 +229,7 @@ integer function fms_register_diag_field_obj &
      call fileptr%set_file_domain(fieldptr%get_domain(), fieldptr%get_type_of_domain())
      call fileptr%init_diurnal_axis(this%diag_axis, this%registered_axis, diag_field_indices(i))
      call fileptr%add_axes(axes, this%diag_axis, this%registered_axis, diag_field_indices(i))
-     call fileptr%add_start_time(init_time)
+     call fileptr%add_start_time(init_time, this%current_model_time)
```

Comment:
Here it's the current model time.  In the routine, it is the start time.

Reply:
This is in the initialization, the current model time is the start_time.